modnet/models.py

```diff
@@ -62,6 +61,8 @@ def __init__(
 
         self.__modnet_version__ = __version__
 
+        if n_feat is None:
```

Comment:
why this exactly?

Reply:
If you're making a model to just do `fit_preset`, you don't really want to specify `n_feat` so `None` is the most appropriate value (rather than leaving it as the default 64).